main/EDPftvarcon.F90

```diff
@@ -1738,6 +1739,17 @@ subroutine FatesCheckParams(is_master)
 
      real(r8) :: sumarea    ! area of PFTs in nocomp mode.
 
+
+     ! Parameters - used to check lmr base rate  r_0 parameter 
+     ! values from Atkin et al., 2017 https://doi.org/10.1007/978-3-319-68703-2_6
+     ! and Heskel et al., 2016 https://doi.org/10.1073/pnas.1520282113
+     real(r8), parameter :: r_1 = 0.2061_r8     ! (umol CO2/m**2/s / (gN/(m2 leaf)))
```

Comment:
Is it possible for `FatesCheckParams` to get these parameter values via a use statement instead of copying them here?

Reply:
Either way? If a goal here is to not expand the scope of this PR to allow for quick bugfix integration, maybe move to FatesConstantsMod for now and we can pull into parameter file in some future PR?

Reply:
I've put them in FatesConstantsMod for now.  We can add them to the parameter file next time a parameter file API update is coming up.